build/secondary/third_party/dart/pkg/meta/BUILD.gn

```diff
@@ -0,0 +1,20 @@
+# THIS FILE WAS BUILT BY HAND BASED ON fuchsia.git EXAMPLES.
```

Comment:
@arbreng -

Any thoughts on whether to keep or change the comment?

Should I add a "TODO" and bug ID?

And/or should I add a copyright block?

Note the fuchsia.git versions don't have a copyright block, but they are auto-generated, and they have only one comment at the top, for example:

```gn
# This file is generated by importer.py for quiver-2.1.5
```